sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -142,3 +141,20 @@ export function getExtendedDateFormat(date: Date): string {
 export function getDateOnlyFormat(date: Date): string {
   return dayjs(date).format(DATE_ONLY_FORMAT);
 }
+
+/**
+ * Get the extended date format(2021 Mar 22) for the date given
+ * @param date date to be retrieved as Extended date format
+ * @returns extended date format like March, 22 2021
+ */
+export function getDateDifferenceInMonth(
+  firstDate: Date | string,
+  secondDate: Date | string,
+): number {
+  if (firstDate && secondDate) {
+    const date1 = dayjs(firstDate);
+    const date2 = dayjs(secondDate);
+    return date1.diff(date2);
+  }
```

Comment:
Will this (**the difference in the 2 dates**) always return a positive result ? Maybe an extra check for negative result would suffice.

Reply:
no, it will return negative results when 2nd date is less than 1st date

Reply:
expected results are -ve, 0 and +ve numbers